demo-private.html

```diff
@@ -44,12 +45,19 @@
 </head>
 <body unresolved>
 
+<google-signin
+  clientId="1054047045356-j8pgqgls9vdef3rl09hapoicumbte0bo.apps.googleusercontent.com"
```

Comment:
The signin component requires a `clientId`.

We have separated the signin component from the `google-sheets` component, so the `google-sheets` doesn't care about the signin at all unless it is `!published`. In the private demo it needs the `google-signin` component to trigger the auth flow, then `google-sheets` uses the `access_token` to talk to the api like it was before.